reconcile/quay_repos.py

```diff
@@ -75,11 +76,15 @@ def fetch_desired_state(quay_api_store):
             continue
 
         for quay_repo in quay_repos:
+            org_name = quay_repo["org"]["name"]
             if not quay_repo["org"]["managedRepos"]:
-                continue
+                logging.error(
+                    f"[{app['name']}] Can not manage repos in {org_name} "
+                    "since managedRepos is set to false."
+                )
+                sys.exit(ExitCodes.ERROR)
```

Comment:
could you continue processing the rest of the repos with the usual `error = True` strategy and then fail at the end?

Reply:
This integration is quite ancient and is not working this way